tests/FTPServiceTest.php

```diff
@@ -24,20 +24,20 @@ public static function setUpBeforeClass()
         Storage::pushService(['ftp' => FTPService::class]);
     }
 
-    protected function setUp()
+    protected function setUp(): void
     {
         $this->ftp_service = Storage::service('ftp');
     }
 
-    protected function tearDown()
+    protected function tearDown(): void
     {
         $this->ftp_service->setConnectionRoot();
     }
 
     public function testUnsecuredConnection()
     {
         $this->assertInstanceOf(FTPService::class, $this->ftp_service);
-        $this->assertInternalType('resource', $this->ftp_service->getConnection());
+        $this->assertIsResource($this->ftp_service->getConnection());
```

Comment:
To upgrade the PHPUnit version to `8`, it should remove the `assertInternalType`. Using the specific type assertions.